notebooks/hello-npu/hello-npu.ipynb

```diff
@@ -0,0 +1,1253 @@
+{
```

Comment:
<p> </p><p>I think this page reference may give additional educational value, could you please integrate link into intro?</p><p><a href="https://docs.openvino.ai/2024/openvino-workflow/running-inference/inference-devices-and-modes/npu-device.html" target="_blank">https://docs.openvino.ai/2024/openvino-workflow/running-inference/inference-devices-and-modes/npu-device.html</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1898/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>